code/datums/components/riding/riding_vehicle.dm

```diff
@@ -201,28 +201,10 @@
 	set_vehicle_dir_offsets(EAST, -18, 0)
 	set_vehicle_dir_offsets(WEST, -18, 0)
 
-
-/datum/component/riding/vehicle/car
-	vehicle_move_delay = 1
-	COOLDOWN_DECLARE(enginesound_cooldown)
-
-/datum/component/riding/vehicle/car/handle_ride(mob/user, direction)
-	. = ..()
-	if(!.)
-		return
-	var/obj/vehicle/sealed/car/car_parent = parent
-	if(!COOLDOWN_FINISHED(src, enginesound_cooldown))
-		return
-	COOLDOWN_START(src, enginesound_cooldown, car_parent.engine_sound_length)
-	playsound(car_parent, car_parent.engine_sound, 100, TRUE)
-
-/datum/component/riding/vehicle/car/clowncar
-	keytype = /obj/item/bikehorn
```

Comment:
u sure about that?

Reply:
The whole point was making it so the clown car doesn't use a riding component, and I tested it thrice, so yes I am sure